db/seeds.rb

```diff
@@ -34,6 +35,16 @@
 
 registered_applications = RegisteredApplication.all
 
+100.times do
+  user = Event.create!(
+    name: Faker::Name.name,
+    registered_application: registered_applications.sample
+)
+end
```

Comment:
Review lines 38 to 43. Remove anything that is unnecessary or that is confusingly named.